.github/PULL_REQUEST_TEMPLATE/feature_template.md

```
@@ -0,0 +1,13 @@
+### Feature Request
```

Markdownlint

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)